controllers/rcj_soccer_referee_supervisor/rcj_soccer_referee_supervisor.py

```diff
@@ -28,10 +31,14 @@ def output_path(
         directory.mkdir(parents=True, exist_ok=True)
 
     p = directory / Path(f'{team_blue}_vs_{team_yellow}-{now_str}')
+    if automatic_mode:
+      match_id = environ.get("MATCH_ID", "")
```

Comment:
All environment variables should be put together on top of file, so we know what we are using.
Also, use 4 spaces, not 2, please.